hub/config/src/main/java/uk/gov/ida/hub/config/application/PrometheusClientService.java

```diff
@@ -12,7 +12,13 @@
 
 public class PrometheusClientService {
     public static final String VERIFY_CONFIG_CERTIFICATE_EXPIRY = "verify_config_certificate_expiry";
+    public static final String VERIFY_CONFIG_CERTIFICATE_EXPIRY_TIMESTAMP = "verify_config_certificate_expiry_timestamp";
     public static final String VERIFY_CONFIG_CERTIFICATE_OCSP_SUCCESS = "verify_config_certificate_ocsp_success";
+    public static final String VERIFY_CONFIG_CERTIFICATE_OCSP_SUCCESS_TIMESTAMP = "verify_config_certificate_ocsp_success_timestamp";
```

Comment:
can this be `verify_config_certificate_ocsp_last_success_timestamp`? otherwise it's not clear what the timestamp means.

Reply:
I have renamed it to `verify_config_certificate_ocsp_revocation_status_last_updated`.

Reply:
`verify_config_certificate_ocsp_revocation_status_last_updated` suggests it will update for any OCSP response, not just successful ones; but I think it's more useful for alerting that we have a metric which only updates when we have a successful OCSP response?

Reply:
We spoke face to face and I linked to the [prometheus best practices for alerting on batch jobs](https://prometheus.io/docs/practices/alerting/#batch-jobs) which feels very similar to this use case. We've agreed to go with `verify_config_certificate_ocsp_last_success_timestamp` 👍